src/BigInteger/AdapterPluginManager.php

```diff
@@ -21,13 +22,23 @@
 class AdapterPluginManager extends AbstractPluginManager
 {
     /**
-     * Default set of adapters
+     * Default set of adapter aliases
      *
-     * @var array
+     * @var string[]
      */
-    protected $invokableClasses = [
-        'bcmath' => 'Zend\Math\BigInteger\Adapter\Bcmath',
-        'gmp'    => 'Zend\Math\BigInteger\Adapter\Gmp',
+    protected $aliases = [
+        'Bcmath' => 'Zend\Math\BigInteger\Adapter\Bcmath',
+        'Gmp'    => 'Zend\Math\BigInteger\Adapter\Gmp',
+    ];
```

Comment:
I'm wondering if we shouldn't also alias `bcmath => Bcmath`, etc to prevent typos?